### PR TITLE
Feature/ab 665 docker latest and docs update

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
updating docker latest and docs

- docker hub latest now updated
  https://hub.docker.com/layers/154540319/riversafe/eze-cli/latest/images/sha256-6b2d3449449e239b3ca5ad6f7f65fcaa8a6ea659817a577f2e8649d1065e11a3?context=repo
- updated unit test as they had fallen below 80%
  (77% -> 80%)
- added license to tool help
  see "eze tools help trufflehog"
- removed duplicated functions from tools.py
  (from ado to github merge and migration)

### CODE REVIEW

File: tests/__test_helpers__/mock_helper.py [hunk 1]
```diff
@@ -83,8 +88,11 @@ def check_installed() -> str:
 
     async def run_report(self, scan_results: list):
         """Method for taking scan results and turning then into report output"""
-
-        return {"title": "test report wip"}
+        print(f"DummyReporter received {len(scan_results)} scans")
+        for scan_result in scan_results:
+            print(f"""[{name_and_time_summary(scan_result, "")}]""")
```
Comment (mgonzales-rs) [inline tests/__test_helpers__/mock_helper.py:93]: just wondering, do we need three **"** ?

File: tests/__snapshots__/core/tool__run_tool-result-output.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "none": 0,
+      "total": 3
+    }
+  },
+  "tool": "Safety",
+  "vulnerabilities": [
+    "NOT UNDER TEST (FROM FIXTURE)"
+  ],
+  "warnings": [
+    "Warning: unpinned requirement 'blackduck' found in C:/dev/repos/sacbdnotify/requirements.txt, unable to check.\nWarning: unpinned requirement 'click' found in tests-integration/__fixture__/python/requirements.txt, unable to check.\n"
```
Comment (mgonzales-rs) [inline tests/__snapshots__/core/tool__run_tool-result-output.json:36]: Its a small query, this is a snapshot, but I noticed it is referring **tests-integration** directory, is it expected?

**Review by mgonzales-rs — APPROVED** [review]

File: tests/core/test_tool.py [hunk 3]
```diff
@@ -155,13 +221,13 @@ def test_normalise_vulnerabilities(self):
...
         # Then
-        output = tool_manager_instance.normalise_vulnerabilities(input_vulnerabilities, input_config)
+        output = tool_manager_instance._normalise_vulnerabilities(input_vulnerabilities, input_config)
 
         output_object = json.loads(json.dumps(output, default=vars))
         expected_output_object = json.loads(json.dumps(expected_output, default=vars))
         assert output_object == expected_output_object
 
-    def test_sort_vulnerabilities(self):
+    def test_test_private__sort_vulnerabilities(self):
```
Comment (mgonzales-rs) [inline tests/core/test_tool.py:230]: perhaps. is this unintentionally test_test ?

File: tests/__snapshots__/core/tool__run_tool-result-output.json (hunk 2 above)
Comment (Anthony-Mckale) [reply tests/__snapshots__/core/tool__run_tool-result-output.json:36]: yes this is expected

File: tests/core/test_tool.py (hunk 3 above)
Comment (Anthony-Mckale) [reply tests/core/test_tool.py]: fixed

**Pull request merged**